code/game/objects/items/weapons/implants/implanter.dm

```diff
@@ -21,10 +21,8 @@
 	. = ..()
 
 /obj/item/implanter/on_update_icon()
-	if (imp)
-		icon_state = "implanter1"
-	else
-		icon_state = "implanter0"
+	. = ..()
+	icon_state = "implanter[imp == TRUE]"
```

Comment:
My beef with using !! is that its easy to misread it and/or consider it a typo. Maybe if we could somehow make a macro for it it would be more readable?